build/scss/scania/variables-dark.scss

```diff
@@ -0,0 +1,150 @@
+/**
+ * Do not edit directly, this file was auto-generated.
+ */
+
+.scania-dark, .tds-mode-dark {
```

Comment:
1. It seems like variables in this file should be part of typography.scss file instead, or?
2. If we still need them, please change class to be ".scania" only as those are typography values.